### PR TITLE
add shannonbradshaw to approvers

### PR DESCRIPTION
**Pull request by shannonbradshaw**
Hello,

Adding myself, shannonbradshaw, to the website approvers list. I've mainly been working on the docs.

I contributed quite a bit to the design, organization, and review of docs for the 1.3 release:
kubeflow/website#2645
kubeflow/website#2655
kubeflow/website#2650
kubeflow/website#2641
kubeflow/website#2638
kubeflow/website#2630
kubeflow/website#2691
kubeflow/website#2963

And am actively involved in the community and docs efforts for the 1.4 release:
kubeflow/kubeflow#6082
kubeflow/website#2986

I led the docs team at MongoDB for several years and am excited to be able to contribute to the Kubeflow docs effort!

-Shannon

### CODE REVIEW

**Comment by jbottum** [comment]
@shannonbradshaw thanks for stepping up to help the Community  /assign @james-jwu

**Review by andreyvelich — COMMENTED** [review]
Thank you for your great contributions @shannonbradshaw!
/lgtm

File: OWNERS [hunk 1]
```diff
@@ -4,6 +4,7 @@ approvers:
   - james-jwu
   - RFMVasconcelos
   - zijianjoy
+  - shannonbradshaw
```
Comment (andreyvelich) [inline OWNERS:7]: Let's keep this list in alphabetical order ?

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by shannonbradshaw** [comment]
@andreyvelich I updated so that approvers are in alphabetical order. Sorry you had to flag that.

**Comment by Arhell** [comment]
сс @Bobgy

**Comment by theadactyl** [comment]
Thanks for your contributions @shannonbradshaw!

**Comment by shannonbradshaw** [comment]
@zijianjoy Thea approved. Can you approve to merge and close this?

**Pull request merged**